simple.css

```diff
@@ -611,4 +611,5 @@ progress::-moz-progress-bar {
 
 progress:indeterminate::-moz-progress-bar {
   background-color: var(--accent-bg);
-}
\ No newline at end of file
+}
```

Comment:
If no CSS changes have been made, better to discard all than submit empty lines.

Dropdown menus could use better styling in header (see: one menu per line issue, could use slightly better design to make it look like a menu)